cmd/gossamer/utils.go

```diff
@@ -72,15 +72,15 @@ func newTestConfig(t *testing.T) *dot.Config {
 
 	cfg := &dot.Config{
 		Global: dot.GlobalConfig{
-			Name:           dot.GssmrConfig().Global.Name,
-			ID:             dot.GssmrConfig().Global.ID,
+			Name:           dot.WestendDevConfig().Global.Name,
```

Comment:
supernit should we assign this to a variable, `westendDevConfig := dot.WestendDevConfig()` to avoid calling the function multiple times?

Reply:
absolutely